Copyboard/Views/Content/CBContentView.swift

```diff
@@ -164,6 +164,46 @@ class CBContentView: CBBaseView {
 			item.animateClickFeedback()
 		}
 	}
+	
+	func performDelete(at indexPaths: [IndexPath]) {
+		let source = filteredItems ?? clipboardItems;
+
+		let objectsToDelete: [CBObject] = indexPaths.compactMap { ip in
+			guard ip.item < source.count else { return nil };
+			
+			return source[ip.item];
+		}
+
+		guard !objectsToDelete.isEmpty else { return };
+
+		if objectsToDelete.count == 1 {
+			// Single item: delete immediately (no alert)
+			_deleteItem(for: objectsToDelete[0]);
+			
+			return;
+		}
+
+		// Multiple items: show confirmation
+		let alert = NSAlert();
+		let count = objectsToDelete.count;
+		alert.messageText = String.localized("Delete %d Items?", arguments: count);
```

Comment:
When adding an alert, theres already one in settings, make sure they both use the same strings for these instead of new ones.